CMake/SlicerCPack.cmake

```diff
@@ -354,7 +354,9 @@ endif()
 # -------------------------------------------------------------------------
 if(CPACK_GENERATOR STREQUAL "NSIS")
 
-  set(Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY "")
+  if(NOT DEFINED Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY)
```

Comment:
```
  # Conditionally set the variable so that a different value
  # can be used to the inner build in the context of custom application.
  if(NOT DEFINED Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY)
```

Reply:
Looks good, thank you!